DXRMapFixups/DeusEx/Classes/DXRFixupM03.uc

```diff
@@ -160,6 +172,14 @@ function PreFirstEntryMapFixes()
             if( m.Name == 'DeusExMover65' ) m.Tag = 'BathroomDoor';
         }
         AddSwitch( vect(3745, -2593.711914, 140.335358), rot(0, 0, 0), 'BathroomDoor' );
+
+        //The Leader can go hostile so easily... just make that not possible
+        foreach AllActors(class'Terrorist',terror){
+            if (terror.BindName=="TerroristLeader"){
+                terror.ChangeAlly('Player',0,True);//Permanently neutral
```

Comment:
fun fact, this guy is t-posing if you ghost into his room, he only activates when you frob his brick